project/pyproject.toml.jinja

```diff
@@ -9,7 +9,11 @@ repository = "https://[[ repository_provider ]]/[[ repository_namespace ]]/[[ re
 homepage = "https://[[ repository_provider ]]/[[ repository_namespace ]]/[[ repository_name ]]"
 keywords = []
 packages = [ { include = "[[ python_package_import_name ]]", from = "src" } ]
-
+classifiers = [
+    "Framework :: VOLTTRON",
+    "Framework :: VOLTTRON :: Agent",
+    "Framework :: VOLTTRON :: 0.1.0"
```

Comment:
@schandrika For the last classifier, do we mean to specify the eventual version of 'volttron-core' that this agent is based on?

Reply:
I would think this should be just documented and left alone. because I assume they would anyway want to come up with a list of versions the agent is compatible with

Reply:
These classifiers won't work